src/components/Navigation/components/Section/Section.tsx

```diff
@@ -112,7 +112,6 @@ export default class Section extends React.Component<Props, State> {
             <span className={styles.Icon}>
               <Icon source={HorizontalDotsMinor} />
             </span>
-            {ariaLabel}
```

Comment:
Just for context, why was the label removed here?

Reply:
Duplicate label on the button container above.